ReactViewControl/ViewModuleContainer.cs

```diff
@@ -52,17 +56,23 @@ public abstract class ViewModuleContainer : IViewModule {
 
         string[] IViewModule.Events => Events;
 
-        string[] IViewModule.DependencyJsSources => DependencyJsSourcesCache.Value;
+        string[] IViewModule.DependencyJsSources => getDependencyJsSources?.Invoke() ?? DependencyJsSourcesCache.Value;
 
-        string[] IViewModule.CssSources => CssSourcesCache.Value;
+        string[] IViewModule.CssSources => getCssSources?.Invoke() ?? CssSourcesCache.Value;
 
         KeyValuePair<string, object>[] IViewModule.PropertiesValues => PropertiesValues;
 
-        void IViewModule.Bind(IFrame frame, IChildViewHost childViewHost) {
+        void IViewModule.Bind(IFrame frame, bool enableHotReload, IChildViewHost childViewHost) {
             frame.CustomResourceRequestedHandler += this.frame.CustomResourceRequestedHandler;
             frame.ExecutionEngine.MergeWorkload(this.frame.ExecutionEngine);
             this.frame = frame;
             this.childViewHost = childViewHost;
+
+            if (enableHotReload) {
```

Comment:
just store the value of the enableHotReload ... as a class field and use that on the GetDependenciesFromEntriesFile no more changes needed

Reply:
btw enableHotReload -> isHotReloadEnabled